client/src/components/StyleBubble.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+
+function StyleBubble(props) {
+  
+  const { style, setSelectedStyle } = props;
+  
+  function handleClick() {
+    console.log(style);
```

Comment:
remove console logs before making your commit
instead write test files to get output from your code
(we are all guilty of this)